tests/Resources/ti.ui.navigationwindow.addontest.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2017-Present by Axway. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+describe.windowsMissing('Titanium.UI.NavigationWindow', _ => {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.navigationwindow.addontest.js;
  Line: 11;
-->

- :warning: tests/Resources/ti.ui.navigationwindow.addontest.js line 11 – '_' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)